internal/server/data/data.go

```diff
@@ -511,6 +513,24 @@ func GlobalCount[T models.Modelable](tx GormTxn, selectors ...SelectorFunc) (int
 	return count, nil
 }
 
+// QuickGlobalCount gives the count of all records, not scoped by org, and not filterable, but very fast.
+func QuickGlobalCount[T models.Modelable](tx GormTxn) (int64, error) {
+	db := tx.GormDB()
+
+	s, err := schema.Parse((*T)(nil), &sync.Map{}, schema.NamingStrategy{})
+	if err != nil {
+		return 0, fmt.Errorf("trying to get table name: %w", err)
+	}
+	tableName := s.Table
```

Comment:
Could we do this without involving gorm, since we are actively working on removing it?

We should be able to pass the table name as an argument, right? 

We could move the queries for `EnqueueHeartbeat` to something like `data.TableCounts` to not expose table names in that case, and expose type specific functions for the two calls in `setupMetrics`.

Reply:
I'd like to be able to get the table name directly from the models. Doesn't appear we're set up for that yet.

Reply:
We do have the table name on some of the unexported aliases `table` types in `data`, but ya, not yet exported anywhere, and most models don't have a `table` equivalent yet.

Reply:
👍🏼 seems like this is fine for now as it will be replaced soon anyway.

Reply:
That's one way to look at it! It'd be nice if new code started using the new style of queries, instead of adding to the number of queries that need to be changed.

For now we could add a `Table()` method to the few models that are used with `QuickGlobalCount`, or create a `map[reflect.Type]string` here to look up the name from the mode type. Either of those should allow this to use regular sql without gorm, right?